CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## [Unreleased]
```

Comment:
As this PR will need to be targeted against the v6 branch, it is easier not to have a changelog entry. Please remove it.

I would add all entries in the v6 branch right before a release.
This avoids git conflicts when rebasing v6 onto main.

Reply:
✔️